components/signup/signupform.tsx

```diff
@@ -259,20 +279,74 @@ export default function SignUpForm() {
 }
 
 
-const PasswordStrengthBar = ({ passwordreset }: {
-    passwordreset: number
-}) => {
+// const PasswordStrengthBar = ({ passwordreset }: {
+//     passwordreset: number
+// }) => {
 
 
-    return (
-        <div className="flex flex-row justify-start gap-2">
-            <div
-                className={
-                    `h-2 w-full rounded-full ${passwordreset >= 1 ? "bg-green-500" : "bg-gray-300"}`
-                }
-            >
+//     return (
+//         <div className="flex flex-row justify-start gap-2">
+//             <div
+//                 className={
+//                     `h-2 w-full rounded-full ${passwordreset >= 1 ? "bg-green-500" : "bg-gray-300"}`
+//                 }
+//             >
+
+//             </div>
+//         </div>
+
+//     )
+// }
+
+function PasswordStrengthBar({ passStrength }: {
+    passStrength: number,
+}) {
+    console.log(passStrength)
+    let barWidth = 0;
+
+    switch (passStrength) {
+        case 0:
+            barWidth = 25;
+            break;
+        case 1:
+            barWidth = 50;
+            break;
+        case 2:
+            barWidth = 75;
+            break;
+        case 3:
+            barWidth = 100;
+            break;
+        default:
+            break;
+    }
 
-            </div>
+
+    const barColor = () => {
+        switch (passStrength) {
+            case 0:
+                return "#EA1111"
+            case 1:
+                return "#FFD700"
+            case 2:
+                return "#00BFFF"
+            case 3:
+                return "#00FF00"
+            default:
+                return "none"
+        }
+    }
+
+    const changeColor = () => ({
+        width: `${barWidth}%`,
+        background: barColor(),
+        height: "7px",
+        borderRadius: "20px",
+        transition: "width 0.5s",
+    })
+    return (
+        <div className="bg-[#444654] rounded-xl h-2" >
```

Comment:
<img width="595" alt="image" src="https://github.com/Abbhiishek/codecommunitymusic-client/assets/86338762/f5e5b495-20b1-4185-b247-5932533d5645">


can you change the bg color to anyone shade from the above, Try to use tailwind class